docs/api-reference/instances/attribute.md

```diff
@@ -0,0 +1,45 @@
+<nav class="fusiondoc-api-breadcrumbs">
+	<a href="../..">Fusion</a>
+	<a href="..">Instances</a>
+</nav>
+
+<h1 class="fusiondoc-api-header" markdown>
+	<span class="fusiondoc-api-icon" markdown>:octicons-key-24:</span>
+	<span class="fusiondoc-api-name">Attribute</span>
+	<span class="fusiondoc-api-pills">
+		<span class="fusiondoc-api-pill-type">special key</span>
+		<span class="fusiondoc-api-pill-since">since v0.3</span>
+	</span>
+</h1>
+
+A [special key](./specialkey.md) which allows you to connect UI objects
+to roblox's attribute system.
```

Comment:
This could be worded more concisely:

"A special key for adding attributes to instances."

Reply:
Have fixed!